pages/Types.html

```diff
@@ -604,6 +605,17 @@ <h2 id="Element"> Element </h2>
 </code></pre>
 <p>You could replace <code>this.value</code> with <code>$(this).val()</code> to access the value of the text input via jQuery, but in that case you wouldn't gain anything.
 </p>
+<h2 id="Text"> Text </h2>
+<p>Text is a node of the Document Object Model (DOM) that represents the textual content of an <a href="#Element">element</a> or an attribute. Consider the following code:
```

Comment:
@kswedberg Could you please have a look. I'm wondering whether there should be a comma before 'or'. Thanks.